iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractTapChangerTest.java

```diff
@@ -364,6 +364,16 @@ public void invalidTargetDeadbandPtc() {
         assertTrue(e.getMessage().contains("2 windings transformer 'twt': Unexpected value for target deadband of phase tap changer: -1.0"));
     }
 
+    @Test
+    public void invalidCurrentLimiterNegativeRegulatingValue() {
+        ValidationException e = assertThrows(ValidationException.class, () -> createPhaseTapChangerWith2Steps(1, 0, true, true,
+                PhaseTapChanger.RegulationMode.CURRENT_LIMITER, -1.0, 1.0, terminal));
+        assertTrue(e.getMessage().contains("phase tap changer in CURRENT_LIMITER mode must have a non-negative regulation value"));
+
+        assertEquals(ValidationLevel.EQUIPMENT, ValidationUtil.checkPhaseTapChangerRegulation(null, PhaseTapChanger.RegulationMode.CURRENT_LIMITER, -1, true, true,
+                terminal, network, ValidationLevel.EQUIPMENT, network.getReportNodeContext().getReportNode()));
```

Comment:
Just calling the `ValidationUtil`'s method here is not a sufficient test. Instead, you can change the minimum validation level then retry to create the tap changer:

```suggestion
        Runnable createTapChanger = () -> createPhaseTapChangerWith2Steps(1, 0, true, true,
                PhaseTapChanger.RegulationMode.CURRENT_LIMITER, -1.0, 1.0, terminal);

        ValidationException e = assertThrows(ValidationException.class, createTapChanger::run);
        assertTrue(e.getMessage().contains("phase tap changer in CURRENT_LIMITER mode must have a non-negative regulation value"));

        network.setMinimumAcceptableValidationLevel(ValidationLevel.EQUIPMENT);
        assertEquals(ValidationLevel.STEADY_STATE_HYPOTHESIS, network.getValidationLevel());
        createTapChanger.run();
        assertEquals(ValidationLevel.EQUIPMENT, network.getValidationLevel());```